unix_integration/Cargo.toml

```diff
@@ -75,7 +75,7 @@ serde_json.workspace = true
 sketching.workspace = true
 
 toml.workspace = true
-tokio = { workspace = true, features = ["rt", "macros", "sync", "time", "net", "io-util"] }
+tokio = { workspace = true, features = ["rt", "rt-multi-thread", "macros", "sync", "time", "net", "io-util"] }
```

Comment:
We don't actually want multi-thread here, this was intentional to keep this as single thread.

Reply:
That should be fixed now.